bioptim/optimization/parameters.py

```diff
@@ -129,6 +144,8 @@ def add(
             The index of the parameter in the parameters list
         penalty_list: Union[Objective, ObjectiveList]
             The objective function associate with the parameter
+        scaling: Float
```

Comment:
Don't think float is supposed to be capitalized, otherwise it won't be recognized by the IDE

Reply:
(Yes the IDE reads this to determine the type, not sure what happens when it conflicts the hints in the function header though... I assume the latter superseeds the docstrings.. Still they should fit)